### PR TITLE
Put 'Validate all policies' before 'Test Policy'

### PR DESCRIPTION
**Pull request by guohaoyu110**
## Related Issue(s)
Close: https://github.com/kyverno/policies/issues/611

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [x] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [x] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

**Comment by guohaoyu110** [comment]
@chipzoller  completed. 
add ignore to the pod-security/enforce/kustomization.yaml

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,40 +14,37 @@ jobs:
...
-        with:
-          go-version: "1.20"
-
-      - name: Test Policy
-        run: go run ./cmd/cli/kubectl-kyverno test ../policies
-        working-directory: kyverno
-    
       - name: Checkout kyverno/policies
-        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+        uses: actions/checkout@v2
```
Comment (chipzoller) [inline .github/workflows/ci.yml:18]: Do not change the checkout version here.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -14,40 +14,37 @@ jobs:
...
           repository: kyverno/policies
-      
+
       - name: Validate all policies
         run: |
           #!/bin/bash
           set -euo pipefail
 
+          # Define the directory to ignore
+          ignore_dir="$GITHUB_WORKSPACE/pod-security/enforce"
```
Comment (chipzoller) [inline .github/workflows/ci.yml:28]: This isn't what I meant. We don't want to ignore a specific directory, we want to exempt any file named `kustomization.yaml`, regardless of where it's located, from being caught by this logic.

**Review by chipzoller — CHANGES_REQUESTED** [review]

**Comment by guohaoyu110** [comment]
@chipzoller  could you please review this updated logic?

**Review by chipzoller — APPROVED** [review]

**Comment by chipzoller** [comment]
Thank you. Once this is merged, please send 3 PRs, each triggering a "bad" result as previously discussed.

**Comment by guohaoyu110** [comment]
> Thank you. Once this is merged, please send 3 PRs, each triggering a "bad" result as previously discussed.

know, will add today.

**Pull request merged**